dpbench/configs/framework_info/numba_mlir.json

```diff
@@ -0,0 +1,10 @@
+{
+    "framework": {
+        "simple_name": "numba_mlir",
+        "full_name": "numba_mlir",
+        "prefix": "mlir",
+        "class": "NumbaMlirFramework",
+        "arch": "cpu",
+        "sycl_device": "level_zero:gpu:0"
```

Comment:
Arch cpu but device gpu? Also for setting up CI we need to run on CPU? Can we run on CPU with Numba-MLIR currently?

Reply:
Removed `arch`,  we are only supporting gpu devices for now, so it won't run on CI yet